### PR TITLE
658

### PR DESCRIPTION
**Pull request by gvn**
Basic contact picker component w. hard coded contact data (don't merge yet! :ghost:).

### CODE REVIEW

File: views/share/index.html [hunk 1]
```diff
@@ -3,37 +3,26 @@
...
-    </li>
-    <li>
-        <div class="cell">
-            <div class="left">
-                <span class="title">Facebook</span>
-                <span class="description">Share on Facebook</span>
             </div>
             <div class="right">
-                <input type="checkbox" class="pull-right">
+                <a href="#" v-on="click: onSMSClick">Add contacts</a>
```
Comment (k88hudson) [inline views/share/index.html]: This needs to be `14px`

File: components/contactPicker/index.less [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+}
+
+.contact-list {
+    color: #6d8c9b;
+    font-weight: 500;
+
+    .letter-header {
+        font-size: 13px;
+        padding: 0.1em 10px;
```
Comment (k88hudson) [inline components/contactPicker/index.less]: vertical padding should be `13px`

File: components/contactPicker/index.less [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    width: 20px;
+    height: 20px;
+    margin: 0 10px 0 0;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &.on {
+        background: #00b5d0;
```
Comment (k88hudson) [inline components/contactPicker/index.less]: @turquoise

File: components/contactPicker/index.less [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        background-color: #fff;
+        padding: 0.5em 10px;
+        display: flex;
+        align-items: center;
+        font-size: 14px;
+    }
+}
+
+.circle-toggle {
+    border: 1px solid #ccc;
```
Comment (k88hudson) [inline components/contactPicker/index.less]: use `@lightGrey`

File: components/contactPicker/index.less [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    font-weight: 500;
+
+    .letter-header {
+        font-size: 13px;
+        padding: 0.1em 10px;
+        border-bottom: 1px solid #e1edf2;
+    }
+
+    .contact {
+        border-bottom: 1px solid #e1edf2;
```
Comment (k88hudson) [inline components/contactPicker/index.less]: use `@lightGrey`

File: components/contactPicker/index.less [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.contact-list {
+    color: #6d8c9b;
+    font-weight: 500;
+
+    .letter-header {
+        font-size: 13px;
+        padding: 0.1em 10px;
+        border-bottom: 1px solid #e1edf2;
```
Comment (k88hudson) [inline components/contactPicker/index.less]: use `@lightGrey`

File: components/contactPicker/index.less [hunk 7]
```diff
@@ -0,0 +1,76 @@
+.contact-picker {
+    background: #F6F7F7;
```
Comment (k88hudson) [inline components/contactPicker/index.less]: @extraGrey

**Comment by k88hudson** [comment]
Looks good, a few design tweaks

File: components/contactPicker/index.less (hunk 2 above)
Comment (gvn) [inline components/contactPicker/index.less]: That seems way too high. This is 13px:

image

Comment (k88hudson) [inline components/contactPicker/index.less]: Oh sorry! I meant the white ones should be 13px

**Comment by thisandagain** [comment]
:+1: Nice!

**Comment by thisandagain** [comment]
What's the status on this @gvn ?

**Comment by gvn** [comment]
I'm waiting for @k88hudson to pull in real contacts so I can hook it up to non dummy data.

File: components/contactPicker/index.less (hunk 2 above)
Comment (gvn) [inline components/contactPicker/index.less]: K...fixed!

File: views/share/index.html [hunk 8]
```diff
@@ -3,37 +3,26 @@
...
             <textarea name="share-message" v-model="shareMessage" class="textarea-large"></textarea>
         </div>
     </div>
 </form>
-<!--
+
 <ul class="list-cell">
-    <lh class="with-border-top">Share To</lh>
+    <lh class="with-border-top">Share Via</lh>
     <li>
```
Comment (k88hudson) [inline views/share/index.html]: Can you actually apply the `cell` class directly to this `li`

Comment (k88hudson) [inline views/share/index.html]: hmmm actually wait let me check this

**Comment by k88hudson** [comment]
@gvn R+

**Pull request merged**